cypress/integration/plugins/dashboards-assistant/conversation_history_spec.js

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { BASE_PATH } from '../../../utils/constants';
+import '@cypress/skip-test/support';
+
+if (Cypress.env('DASHBOARDS_ASSISTANT_ENABLED')) {
+  describe('Assistant conversation history spec', () => {
+    before(() => {
+      // Set welcome screen tracking to false
+      localStorage.setItem('home:welcome:show', 'false');
+      // Hide new theme modal
+      localStorage.setItem('home:newThemeModal:show', 'false');
```

Comment:
do we want to get their current value first and then set the back to the original value after these tests are complete in case there are tests specific to these values.